docs/getting-started/ea-req.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[ea-requirements]]
+= Entity Analytics prerequisites
+
+[discrete]
+== Privileges
+
+To enable the risk scoring engine, you need the following privileges:
+
+Cluster privileges:
+
+* `manage_index_templates`
+* `manage_transform`
+
+Index privileges:
+
+`all` privilege for `risk-score.risk-score-*`
+
+{kib} privileges:
+
+* **All** for the **Saved Objects Management** feature under **Management**
+* **Read** for the **Security** feature 
+
+[discrete]
+== {es} resource guidelines
+
+Follow these guidelines to ensure clusters have adequate memory to handle data volume:
+
+* With 2GB of Java Virtual Machine (JVM) heap memory, the risk scoring task can safely process around 44 million documents (30 days of risk data with an ingest rate of 1000 documents per minute).
+
+* With 1GB of JVM heap, the risk scoring task can safely process around 20 million documents (30 days of risk data with an ingest rate of around 450 documents per minute).
+
+[discrete]
+== Known limitations
+
+* You can only enable the risk scoring engine in a single {kib} space within a cluster.
+
+* The risk scoring engine uses the internal {kib} user to score all hosts and users. This means the scoring task does not respect custom user or role permissions. All alerts from the configured {kib} space will contribute to the entity's risk.
```

Comment:
```suggestion
* The risk scoring engine uses the internal {kib} user to score all hosts and users, and doesn't respect privileges applied to custom users or roles. All alerts from the configured {kib} space will contribute to host and user risk scores.
```

Reply:
One more quick note - I'm not sure if we need to explain what the "configured Kibana space" is, or if it's evident that we're talking about the Kibana space that has the risk score engine enabled. If we do, here's an potential revision:

```suggestion
* The risk scoring engine uses the internal {kib} user to score all hosts and users, and doesn't respect privileges applied to custom users or roles. After you turn on the risk scoring engine for a {kib} space, all alerts in the space will contribute to host and user risk scores.
```

Reply:
@nastasha-solomon I like your elaboration on the contribution of alerts from the Kibana space. Thanks.